src/Package/Impl/DataInspect/VariableGridWindowPane.cs

```diff
@@ -24,5 +24,10 @@ internal sealed class VariableGridWindowPane : RToolWindowPane {
             }
             _gridHost.SetEvaluation(evaluation);
         }
+
+        protected override void OnClose() {
+            base.OnClose();
+            _gridHost.CleanUp();
```

Comment:
Please verify that this code is called when user closes the tab and is not called when user toggles between tabs